File: Python-Gemini/models/ticket.py
from dataclasses import dataclass
from datetime import datetime

@dataclass
class Ticket:
    """이용권 정보를 담는 데이터 클래스"""
    ticketid: int
    purchase_time: datetime
    expire_time: datetime
    status: str
    userid: int
    username: str # 리스트 출력 시 JOIN된 사용자 이름을 담기 위해 포함

File: C#-Original/Model/Ticket.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Security.Policy;
using System.Text;
using System.Threading.Tasks;

namespace SharingTm.Model
{
    public class Ticket 
    {
        public int ticketID { get; set; }
        public int userID { get; set; }
        public string userName { get; set; } // 출력용
        public DateTime purchaseTime { get; set; }
        public DateTime expireTime { get; set; }
        public string status { get; set; }
    }
}
